lib/msal-browser/apiReview/msal-browser.api.md

```diff
@@ -1579,6 +1611,23 @@ export { ServerError }
 
 export { ServerResponseType }
 
+// Warning: (ae-missing-release-tag) "SessionStorage" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export class SessionStorage implements IWindowStorage<string> {
```

Comment:
I like this. Helps us with more flexibility between storage options instead of a binary choice.